src/pages/Integrations/SplunkSetup/Constants.ts

```diff
@@ -0,0 +1,23 @@
+export const RBAC_GROUPS_API = '/api/rbac/v1/groups/';
+export const RBAC_ROLES_API = '/api/rbac/v1/groups/{}/roles/';
+export const RBAC_ADD_USER_API = '/api/rbac/v1/groups/{}/principals/';
```

Comment:
I'm going to add some RBAC API files on a separate PR - i'll let you know when they are in